instance/tests/integration/integration_instance.py

```diff
@@ -164,20 +164,38 @@ def assert_mongo_db_provisioned(self, instance):
             self.assertTrue(result)
             db.logout()
 
+    def assert_lms_users_provisioned(self, user, appserver):
+        """
+        Ensure the lms user playbook was run on the appserver.
+        """
+        self.assertEqual(appserver.lms_users.count(), 1)
+        self.assertEqual(appserver.lms_users.get(), user)
+        self.assertTrue(appserver.lms_user_settings)
+
+        lms_user_playbook = appserver.lms_user_creation_playbook()
```

Comment:
@pomegranited What does `lms_user_creation_playbook` do? The name seems a little unusual for a method. Maybe it should be a property?

Just curious, no need to change anything in the scope of this PR.